__tests__/__integration__/imperative.integration.test.ts

```diff
@@ -0,0 +1,226 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { runCliScript, stripNewLines } from "../__src__/TestUtils";
+import { TestEnvironment } from "../__src__/environment/TestEnvironment";
+import { ITestEnvironment } from "../__src__/environment/doc/response/ITestEnvironment";
+import { ITestPropertiesSchema } from "../__src__/properties/ITestPropertiesSchema";
+
+let testEnvironment: ITestEnvironment;
+let host: string;
+let port: number;
+let user: string;
+let pass: string;
+let systemProps: ITestPropertiesSchema;
+
+describe("imperative create profile", () => {
+
+    // Create the unique test environment
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "zos_check_status"
+        });
+
+        systemProps = testEnvironment.systemTestProperties;
+        host = systemProps.zosmf.host;
+        port = systemProps.zosmf.port;
+        user = systemProps.zosmf.user;
+        pass = systemProps.zosmf.pass;
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    describe("create zosmf profile", () => {
+
+        // Create a separate test environment for no profiles
+        let TEST_ENVIRONMENT: ITestEnvironment;
+        let SYSTEM_PROPS: ITestPropertiesSchema;
+
+        beforeAll(async () => {
```

Comment:
Is the `beforeAll` and `afterAll` at this level required? Since the TestEnvironment isn't being used to create a profile, maybe the one from line 28 could be reused.

Reply:
This is not required, and they are all using the same testEnvironment now.